modelling/src/test/java/org/axonframework/modelling/command/inspection/AnnotatedAggregateMetaModelFactoryTest.java

```diff
@@ -256,6 +296,24 @@ void testFindJavaxPersistenceIdentifier() {
         assertEquals("id", inspector.routingKey());
     }
 
+    @Test
+    void testFindJavaxPersistenceGetterIdentifier() {
+        AggregateModel<JavaxPersistenceGetterAnnotatedHandlers> inspector =
+                AnnotatedAggregateMetaModelFactory.inspectAggregate(JavaxPersistenceGetterAnnotatedHandlers.class);
+
+        assertEquals("id", inspector.getIdentifier(new JavaxPersistenceGetterAnnotatedHandlers()));
+        assertEquals("getId", inspector.routingKey());
+    }
+
+    @Test
+    void testFindJavaxPersistenceMethodIdentifier() {
+        AggregateModel<JavaxPersistenceMethodIdAnnotatedHandlers> inspector =
+                AnnotatedAggregateMetaModelFactory.inspectAggregate(JavaxPersistenceMethodIdAnnotatedHandlers.class);
+
+        assertEquals("id", inspector.getIdentifier(new JavaxPersistenceMethodIdAnnotatedHandlers()));
+        assertEquals("calculatedId", inspector.routingKey());
```

Comment:
This case, when users only provide an `@Id` field, is obviously rather problematic on a method. Not sure we can do anything about this...